examples/3_write_serial.py

```diff
@@ -7,22 +7,22 @@
 License: LGPLv3+
 """
 import openPMD
-# import numpy as np
+import numpy as np
 
 
 if __name__ == "__main__":
     # user input: size of matrix to write, default 3x3
     size = 3
 
     # matrix dataset to write with values 0...size*size-1
-    # global_data = np.range(size*size, dtype=np.double)
+    global_data = np.arange(size*size, dtype=np.double)
 
     print("Set up a 2D square array ({0}x{1}) that will be written".format(
         size, size))
 
     # open file for writing
     series = openPMD.Series(
-        "../samples/3_write_serial.h5",
+        "../samples/3_write_serial_py.h5",
```

Comment:
As this is supposed to be run on all platforms, we probably should use [```os.path.join```](https://docs.python.org/3.6/library/os.path.html#os.path.join). This has already come up as a problem in #229.

Reply:
Good catch!
Huh, I wonder why it passes the AppVeyor tests...

Reply:
HDF5 triggers a [warning](https://ci.appveyor.com/project/ax3l/openpmd-api/build/1.0.211/job/og7t29wucynkm1r7?fullLog=true#L2831), but does not abort.
That is because the ```ASSERT``` in the backend that would catch it is a NOOP (will be fixed in #229).